IPython/core/application.py

```diff
@@ -202,12 +202,8 @@ def initialize_subcommand(self, subc, argv=None):
             import time
             self.log.warning("Subcommand `ipython {sub}` is deprecated and will be removed "
                              "in future versions.".format(sub=subc))
-            self.log.warning("You likely want to use `jupyter {sub}`... continue "
-                             "in 5 sec. Press Ctrl-C to quit now.".format(sub=subc))
-            try:
-                time.sleep(5)
-            except KeyboardInterrupt:
-                sys.exit(1)
+            self.log.warning("You likely want to use `jupyter {sub}`... in the"
+                             "future".format(sub=subc))
```

Comment:
The message will end 'thefuture', because there's no space between the words, I think. Anyone want to do a quick extra PR?

Also, I think the `...` is probably superfluous now.